### PR TITLE
Added MEMORY_GROWTH_STEP setting.

### PR DESCRIPTION
**Pull request by MarcelKlammer**
It allows for a slower memory growth and grows it at a fixed rate rather than doubling it.

### CODE REVIEW

File: src/library.js [hunk 1]
```diff
@@ -582,11 +582,16 @@ LibraryManager.library = {
 
     // TODO: see realloc_buffer - for PTHREADS we may want to decrease these jumps
     while (newSize < requestedSize) { // Keep incrementing the heap size as long as it's less than what is requested.
+#if MEMORY_GROWTH_STEP != -1
+      // Memory growth is fixed to a multiple of the WASM page size of 64KB (eg. 16MB) set be the user.
```
Comment (kripken) [inline src/library.js]: typo, `be => by`.

**Comment by MarcelKlammer** [comment]
I based the test on the WASM_MEM_MAX test, stepping up by 1MB increments to 130MB max memory.

**Comment by kripken** [comment]
Looks good aside from failures on CI - seems it fails a little before the expected point. I wonder if that's because of existing allocations? Might be enough to see that the test verifies memory grows by 1MB each time, and that it can do over 100 before failing in the 100-130 range.

**Comment by MarcelKlammer** [comment]
It's an exact calculations. The TOTAL_STACK wasn't accounted for. 
So I set TOTAL_STACK to 1Mb and now it's stopping exactly where it should be.

That should be all from my side?

**Comment by kripken** [comment]
Great, thanks @MarcelKlammer!

**Pull request merged**